### ISSUE TITLE
No System partition mounted

### ISSUE THREAD

**Issue by hamzaarbeed** [Meetoul/twrp_device_Unihertz_Jelly2#2]
Hello, I tried to flash something. but when I go to wipe, advanced wipe . I cannot see system partition.
I entered the unlock pin and I allowed system modifications. 
When I see the internal storage from a PC. The files and folders name look encrypted. they are non sense.

**Comment by Meetoul** [comment]
Thank you for your feedback.

I know about the issue with system partition mounting. I'll check it soon. Btw, you do not need to wipe system partition before installing something. Wiping Dalvik, Cache, Data and Internal storage is enough.

Decryption of FBE encrypted files in recovery is a huge pain. Sometimes it works, sometimes not, probably it depends on ROM or something. Decryption fails because of errors in TEE, but this part is completely proprietary, so it's very hard to debug. I've already spent about 3 days trying to get data decryption working stably and I haven't gave up yet, but I cannot guarantee that I'll fix it someday.

**Comment by hamzaarbeed** [comment]
I didn't flash recovery the correct way. I flashed it through official twrp app. I believe that's why I couldn't  clear cach or data or install linage os.
I flashed it in fastboot, and it is working fine.
I flashed lineage os 18.1. Thank you so much.
The only thing. There is a notification says "Serial console enabled Performance is impacted. To disable, check bootloader".
How to remove it. is it something I need to turn off
